core/database/foxx/api/coll_router.js

```diff
@@ -954,7 +1128,26 @@ router
                     res.send({});
                 },
             });
+            logger.logRequestSuccess({
+                client: client?._id,
+                correlationId: req.headers["x-correlation-id"],
+                httpVerb: "GET",
+                routePath: basePath + "/move",
+                status: "Success",
+                description: "Move items from source collection to destination collection",
```

Comment:
Source and and dest are required parameters these would be helpful to include in the description.